pos_deposit/views/product_template.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='options']" position="inside">
```

Comment:
I'm not sure this position is the better one.

![image](https://github.com/user-attachments/assets/7b78f141-7bd1-4488-985d-95a8d6822020)

- label of the field is not displayed.
- helper doesn't looks to be visible

I propose to put it in the "sale" part, around available in pos.

![image](https://github.com/user-attachments/assets/376ea72a-024b-4593-b142-3d90ded2134d)

(also make it invisible if product == service makes sense IMO).